src/App.js

```diff
@@ -69,26 +69,32 @@ const App = () => {
             options={options}
         /> */}
 
-        <Routes>
-          <Route 
-            exact 
-            path="/" 
-            element={
-              <ProtectedRoute>
-                <HomePage />
-              </ProtectedRoute>
-            } 
-          />
-          <Route  
-            path="/login" 
-            element={<LoginPage />} 
-          />
-      </Routes> 
-      </InputContext.Provider>  
-      <Footer />
-    </>
+                <Routes>
+                    <Route
+                        exact
+                        path="/"
+                        element={
+                            <ProtectedRoute>
+                                <HomePage />
+                            </ProtectedRoute>
+                        }
+                    />
+                    <Route
+                        path="/login"
+                        element={
+                            //Uncomment protected route when authentication is done
 
-  );
+                            // <ProtectedRoute requiredAuthLevel='anonymous'>
+                            <NewLogin />
+                            // </ProtectedRoute>
+                        }
+                    />
+                    <Route path="/sign-up" element={<NewSignUp/>} />
```

Comment:
Can you add the commented out ProtectedRoute the same as you did for the NewLogin Page here?